docs/specs/sections/protocol.tex

```diff
@@ -0,0 +1,105 @@
+
+In Citadel, each party involved in the protocol keeps static keys, as we detail now. Let $G, G'\gets\mathbb{J}$ be two generators for the subgroup $\mathbb{J}$ of order $t$ of the Jubjub. The keys of each party as the following.
+
+\begin{itemize}%[label=$\bullet$]
+	\item \textit{Secret key:} $\sk = (a,b)$, where $a,b\gets \F_t$.
+	\item \textit{Public key:} $\pk = (A,B)$, where $A = a G$ and $B = b G$.
+\end{itemize}
+
+The workflow of the Citadel protocol is depicted in Figure \ref{fig:protocol}, and described with full details as follows.
+
+\begin{figure}[h]
+	\centering
+		\includegraphics[width=390pt,draft=false]{images/protocol.eps}
+	\caption{Overview of the protocol messages exchanged between the user, the Dusk Network, and the SP.}
+	\label{fig:protocol}
+\end{figure}
+
+\begin{enumerate}
+	\item (\textbf{user}) $\mathsf{request\_license}$ : compute a license stealth address $(\lpk, R_{\lic})$ belonging to the user, using the user's own public key, as follows.
```

Comment:
I am missing the computation of `lpk'` in the protocol. Should it be computed here? 

I am particularly interested because in one of my last comments you said that the SP has the opening of the session_id, so they can check if `c` is correct, and I would like to understand how lpk' is computed and why SP has access to it.

Reply:
the SP has the opening to com_3, to check if the value `c` was correctly set.